drivers/i3c/master/mipi-i3c-hci/core.c

```diff
@@ -471,7 +471,7 @@ static int i3c_hci_send_ccc_cmd(struct i3c_master_controller *m,
 	    ccc->rnw, ccc->dbp, ccc->db, ccc->ndests,
 	    ccc->dests[0].payload.len);
 
-	if (!i3c_device_power) {
+	if (!i3c_device_power && m->bus.context == I3C_BUS_CONTEXT_JESD403) {
```

Comment:
since this workaround , can you add note saying this also should reverted as part of workaround removal.

Reply:
> since this workaround , can you add note saying this also should reverted as part of workaround removal.

I can.